### ISSUE TITLE
Feature Request: Blank Screen

### ISSUE THREAD

**Issue by ghplw** [Ylianst/MeshCentral#416]
This has been raised before but I am unable to reopen the original thread myself.

https://github.com/Ylianst/MeshCentral/issues/54

The original thread discussed that it is possible on some AMT firmwares but not in the agent.

RealVNC has this feature and TightVNC used to have this feature but it works by using power management features to turn of the screen so it is hardware dependent. I can't think of any hardware I have out there that wouldn't support this so it would be a nice feature to have.

Some other programs achieve this by installing a custom driver. I'm not sure how they work but maybe it could be a a dummy monitor and the agent tells the computer to switch all output to that dummy monitor and then switches it back again on disconnect or when the agent senses that the connection has been lost?

**Comment by Ylianst** [comment]
This is useful for digital signs and other usages. Bryan will have to take a look at this one. Given that we are swamped with requests already, I will put this as low priority for now.

**Comment by ckibodeaux** [comment]
I too would like to see screen blanking as a possible feature. I've read the other posts #2550 #54 and would like to share our use case.

TL;DR - We deal with some very annoying users that don't understand phone calls, banner notices that a technician is connected, or why their keyboard/mouse won't move and end up causing headaches for technicians.

As an MSP focusing on the hospitality industry (hotels, motels, etc.) we do a lot of remote maintenance/updates/configs/etc of machines. It has been our experience that hotel clerks don't understand the small banner showing an agent is connected (they "x" it closing the session), when they see a window open that they don't know they cancel it causing installs/updates to fail, & when the keyboard/mouse is locked they think the computer is frozen and do a hard reboot. When we used another software (ConnectWise Control, previously ScreenConnect) we were able to black out the screen and display a large custom message such as "A technician is currently working on this machine, if you need access please contact the help desk at phone xxx-xxxx."

We have been working with managers to get users to understand, but staff turnaround is so high by the time the staff is "trained" not to interfere with the remote work, they quit and the cycle starts again. We have called sites to inform the user that maintenance is taking place to use another system until it is complete and still we have problems with users interfering. Not all sites are this bad, but we have had some doozies (technical term :P ).

Thanks for reading and we vote to support this feature.

**Comment by NickRomanek** [comment]
We'd love to have this feature too, it is the only thing preventing us from using the software. We lease a device out to customers (the device has a 'special sauce' of software) so when we remote into the device to perform the work, they cannot see the software we're using. Thank you for building an amazing product, I understand this feature isn't geared towards your intended audience.

**Comment by si458** [comment]
start to implement this,
you can now use `display sleep` and `display awake` to turn the screen on/off inside the device console tab 
(code was already built into agent just not in web front end) https://github.com/Ylianst/MeshCentral/commit/08d5aef9c9bb381ff5168106586c7808e86f70cb
i now need to work out how to disable the mouse movements now!

**Comment by si458** [comment]
you can also disable keyboard/mouse movements using the 'keyboard' icon in the bottom right hand side
but sadly, when you move the mouse on the screen, the monitor wakes up?
so need to figure out HOW to stop this

**Comment by MaartenNano** [comment]
Any news on this issue? I would like to see something like this integrated if at all possible.
